Portfolio.html

```diff
@@ -4,11 +4,85 @@
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="Portfolio.css">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Media-Query.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rob Reeves Portfolio</title>
 </head>
-<body>
-
-
 
+<div>
+    <nav class="main-nav">
+        <div class="nav-button-left">
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </nav>
+        <div class="main-header-wrapper">
+            <div class="main-header-centre">
+                <h1>ROB REEVES</h1>
+                    <div class="sub-heading-wrapper">
+                        <div class="sub-heading-rule-before"></div>
+                            <p class="sub-heading">THIS WEBSITE IS ALL ABOUT ME!</p>
+                        <div class="sub-heading-rule-after"></div>
+                    </div>
+                    <div class="sub-heading-wrapper2">
+                        <p class="sub-heading2">There are lots of interesting things about me on this website. Please take your time and enjoy your stay. If you have any queries about anything then please don't hesitate to drop me a line.</p>
+                    </div>
+            </div>
+        </div>
+    <div class="main-content" id="C1">
```

Comment:
Consider using a section tag in place of div

Reply:
done.